src/gpodder/util.py

```diff
@@ -1009,7 +1009,10 @@ def get_first_line(s):
     Returns only the first line of a string, stripped so
     that it doesn't have whitespace before or after.
     """
-    return s.strip().split('\n')[0].strip()
+    try:
```

Comment:
Can you do it as this?
```
if s:
    return s.strip().split('\n')[0].strip()
return ''
```
so there is no blind `except:`

Reply:
Updated.